### PR TITLE
CSI-2350 finish allowing openssl 1.0.2

### PR DESCRIPTION
**Pull request by oriyarde**
a continuation of https://github.com/IBM/ibm-block-csi-driver/pull/249

### CODE REVIEW

File: Dockerfile-csi-controller [hunk 1]
```diff
@@ -33,6 +33,8 @@ COPY controller/requirements.txt /driver/controller/
 RUN pip3 install --default-timeout=100 --upgrade pip==19.1.1
 # avoid default boringssl lib, since it does not support z systems
 ENV GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=True
+# TODO: remove CRYPTOGRAPHY_ALLOW_OPENSSL_102 when upgrading to ubi8
```
Comment (ArbelNathan) [inline Dockerfile-csi-controller:36]: is there any ticket for it that you can link?

Comment (oriyarde) [reply Dockerfile-csi-controller:36]: no, and AFAIK it might take years before we want to do that, so I think it's too early to open a ticket

Comment (ArbelNathan) [reply Dockerfile-csi-controller:36]: so it will be a legacy TODO?

Comment (oriyarde) [reply Dockerfile-csi-controller:36]: you could call it that. it's just something that needs to be done when changing the base image to ubi8 in this file

**Review by roysahar-ibm — APPROVED** [review]

**Review by ArbelNathan — APPROVED** [review]

**Pull request merged**